docs/notebooks/topic_labeling.ipynb

```diff
@@ -0,0 +1,125 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**initial setup**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import gensim\n",
+    "import os\n",
+    "import itertools\n",
+    "\n",
+    "wiki_path = 'D:\\gensim_data\\simplewiki'\n",
+    "wiki_prefix = 'gensim_'\n",
+    "\n",
+    "# load id->word mapping (the dictionary), one of the results of step 2 above\n",
+    "id2word = gensim.corpora.Dictionary.load_from_text(os.path.join( wiki_path, wiki_prefix + 'wordids.txt.bz2'))\n",
+    "\n",
+    "# load corpus iterator\n",
+    "mm = gensim.corpora.MmCorpus(os.path.join( wiki_path, wiki_prefix + 'tfidf.mm'))\n",
+    "\n",
+    "# calculate LDA\n",
+    "lda = gensim.models.ldamodel.LdaModel(corpus=mm, id2word=id2word, num_topics=400)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**get_topic_labels function**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Algorithm parameters\n",
```

Comment:
More description (what, why, how it works) is always a good idea

Reply:
To achieve that, I will be basically coping into comments the explanations I've put into my Topic Labeling Methods doc. Hope it's OK